credentialz/credentialz.proto

```diff
@@ -195,6 +195,10 @@ service Credentialz {
 
   // An RPC to ask a target if it can generate a Public/Private key pair
   rpc CanGenerateKey(CanGenerateKeyRequest) returns (CanGenerateKeyResponse);
+
+  // Get the current public keys from the host. Each configured key will be
+  // returned in the provided list.
+  rpc GetPublicKeys(GetPublicKeysRequest) returns (GetPublicKeysResponse);
```

Comment:
I wonder if this could be made yet another `oneof` message in the `Rotate` stream.

Reply:
up to you?